spyder/config/main.py

```diff
@@ -73,7 +73,11 @@
               'single_instance': True,
               'open_files_port': OPEN_FILES_PORT,
               'tear_off_menus': False,
+              'normal_screen_resolution': True,
               'high_dpi_scaling': False,
+              'high_dpi_custom_scale_factor': False,
+              'high_dpi_custom_scale_factors': '2;',

```

Comment:
A float number could cause glitches, as said in the second question of this [blog](https://blog.qt.io/blog/2016/01/26/high-dpi-support-in-qt-5-6/)